hr_employee_top_department/models/hr_employee.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import _, api, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = ["hr.employee"]
+
+    top_dept_id = fields.Many2one(
+        "hr.department",
+        string="Top Department",
+        compute="_compute_top_department",
+        help="Company",
+    )
+
+    @api.depends("department_id")
```

Comment:
No need to define 'depends' if top_dept_id is not stored, although it does not do any harm if we keep it.